.github/workflows/action-build.yml

```diff
@@ -41,13 +48,13 @@ jobs:
           make dist-${{ inputs.runtime }}
           # Check if there's any files to archive as tar fails otherwise
           if stat dist/bin/* >/dev/null 2>&1; then
-            tar -czf dist/containerd-shim-${{ inputs.runtime }}-${{ inputs.os }}.tar.gz -C dist/bin .
+            tar -czf dist/containerd-shim-${{ inputs.runtime }}-${{ inputs.slug }}.tar.gz -C dist/bin .
```

Comment:
why can't we continue to use `os` here?

Reply:
I didn't want to encode `ubuntu-XX.YY` into the artifacts since we generate a `musl` static binary that can run in any distro.

I'd also like to evolve this to be `linux-musl` and `linux-gnu`. As it stands, we are loosing coverage of `glibc` builds, and I would like to avoid that.